README.md

```diff
@@ -40,6 +42,7 @@ To run a training, use the following command:
 ```bash
 python tools/main.py --config-file configs/EfficientNetV2_small_gcn.yml --gpu-num 1 custom_datasets.roots "['<data_root>/train.json', '<data_root>/train.json']" data.save_dir <work_dir>
 ```
+You can also run estimation of the optimal learning rate value prior to training. To anable this step, append `lr_finder.enable True` to the previous command.
```

Comment:
hmmm, we didn't test it. Are you sure to keep this instruction?

Reply:
I've tried, the training works with the found LR. Can this LR be totally wrong?

Reply:
I haven't tested it. With multilabel classification I launched separate script with optuna. But I think it won't be totally wrong, but the speed of lr searching maybe high since the resolution isn't changing.